src/storage/mvcc/write.rs

```diff
@@ -149,6 +179,15 @@ impl Write {
                 .map(|v| *v == PROTECTED_ROLLBACK_SHORT_VALUE)
                 .unwrap_or_default()
     }
+
+    #[inline]
+    pub fn to_owned(&self) -> Write {
```

Comment:
I think this should be an implementation of ToOwned, rather than an inherent method.

Reply:
Borrow cannot be implemented so that ToOwned cannot be implemented as well..

Reply:
:-(